xamarin/Xamarin.Forms
# [iOS] Label HorizontalTextAlignment="Center" not working in conjunction with LineHeight on iOS

Label HorizontalTextAlignment="Center" not working in conjunction with LineHeight on iOS

### Description of Change ###

Changed span.ToAttributed method to set the text alignment for the attributed text in iOS.

### Issues Resolved ### 
- fixes #4262 

### API Changes ###
None

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###
In iOS, when Horizontal Text Alignment and Line Height both set now will do the alignment correctly on label.

### Before/After Screenshots ### 
Before
![previous](https://user-images.githubusercontent.com/18006944/47823639-0f083e00-ddce-11e8-9a9b-cb30528f3c6f.PNG)

After 
![after](https://user-images.githubusercontent.com/18006944/47823641-13345b80-ddce-11e8-9af9-e13e846a6f38.PNG)

### Testing Procedure ###
Start ControlGallery.iOS, Search and go to G4262 Page in tests screen, Two labels should both be centered.
